runtime/vam/expr/genarithfuncs.go

```diff
@@ -77,13 +77,13 @@ func genFunc(name, op, typ string, lhs, rhs vector.Form) string {
 		} else {
 			s += fmt.Sprintf("val := super.New%s64(lconst %s rconst)\n", typ, op)
 		}
-		s += "return vector.NewConst(val, lhs.Len(), nil)\n"
+		s += "return vector.NewConst(val, lhs.Len(), vector.Or(l.Nulls, r.Nulls))\n"
```

Comment:
Nit: Instead of repeating this, up above change

```go
	s += genVarInit("l", typ, lhs)
	s += genVarInit("r", typ, rhs)
```

to

```go
	s += genVarInit("l", typ, lhs)
	s += genVarInit("r", typ, rhs)
	s += "nulls := vector.Or(l.Nulls, r.Nulls)\n"
```

and then just do this here (and in the two spots with `vector.Or` below):

```go
		s += "return vector.NewConst(val, lhs.Len(), nulls)\n"
```